content/sensu-go/5.18/api/overview.md

```diff
@@ -420,6 +416,20 @@ curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/co
 
 ### Examples
 
+#### Values with special characters
+
+To use a label or field selector with string values that include special characters like hyphens and underscores, place the value in single or double quotes:
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
```

Comment:
```suggestion
curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" -X GET http://127.0.0.1:8080/api/core/v2/entities -G \
```

Reply:
* curl shows `HTTP/1.1 405 Method Not Allowed` unless I explicitly specify the `GET` verb and 
* I think `region` is a better fit as an entity label